### PR TITLE
LPS-177633 add validation for negative values

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**
[LPS-177633](https://issues.liferay.com/browse/LPS-177633)

**Steps to reproduce:**
1. Set the Maximum Number of Pages in the system setting (e.g. set 200 as the maximum number)
2. Navigate to the site setting
3. Choose the PDF Preview site scope
4. Configure the number of pages as -1

- LPS-177633 add validation for negative numbers on command
- LPS-177633 add min value to the input
- LPS-177633 add aria-label to input

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
@ambrinchaudhary I put the aria label on the last commit because I saw this on other `ClayInput`. Please feel free to change the message or deleting it if not necessary

Thanks!!!

**Review by ambrinchaudhary — APPROVED** [review]

File: modules/apps/document-library/document-library-preview-pdf/src/main/java/com/liferay/document/library/preview/pdf/internal/portlet/action/EditPDFPreviewMVCActionCommand.java [hunk 1]
```diff
@@ -68,9 +68,15 @@ protected void doProcessAction(
 		}
 
 		try {
+			int maxNumberOfPages = ParamUtil.getInteger(
+				actionRequest, "maxNumberOfPages");
+
+			if (maxNumberOfPages < 0) {
+				throw new PDFPreviewException();
```
Comment (adolfopa) [inline modules/apps/document-library/document-library-preview-pdf/src/main/java/com/liferay/document/library/preview/pdf/internal/portlet/action/EditPDFPreviewMVCActionCommand.java:75]: Can you add a simple error message with the failure reason? E.g. "Maximum number of pages must be a non-negative integer".

**Review by adolfopa — CHANGES_REQUESTED** [review]

File: modules/apps/document-library/document-library-preview-pdf/src/main/java/com/liferay/document/library/preview/pdf/internal/portlet/action/EditPDFPreviewMVCActionCommand.java (hunk 1 above)
Comment (AliciaGarciaGarcia) [reply modules/apps/document-library/document-library-preview-pdf/src/main/java/com/liferay/document/library/preview/pdf/internal/portlet/action/EditPDFPreviewMVCActionCommand.java:75]: Do you think it's necessary?
The validation will be done on the front-end part, 
This is a fallback (guarding our backs... pun intended) value on the "exceptional" case that skips the front.

Comment (adolfopa) [reply modules/apps/document-library/document-library-preview-pdf/src/main/java/com/liferay/document/library/preview/pdf/internal/portlet/action/EditPDFPreviewMVCActionCommand.java:75]: I think it is useful. It takes almost no effort and gives some context on what the error is about if it shows up later in some log file.

**Comment by AliciaGarciaGarcia** [comment]
Normal case:
image


On the case the front is skiped 
image

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

File: modules/apps/document-library/document-library-preview-pdf/src/main/java/com/liferay/document/library/preview/pdf/internal/portlet/action/EditPDFPreviewMVCActionCommand.java [hunk 2]
```diff
@@ -68,9 +68,17 @@ protected void doProcessAction(
 		}
 
 		try {
+			int maxNumberOfPages = ParamUtil.getInteger(
+				actionRequest, "maxNumberOfPages");
+
+			if (maxNumberOfPages < 0) {
+				throw new PDFPreviewException(
+					"maximum-number-of-pages-limit-must-be-a-non-negative-" +
```
Comment (adolfopa) [inline modules/apps/document-library/document-library-preview-pdf/src/main/java/com/liferay/document/library/preview/pdf/internal/portlet/action/EditPDFPreviewMVCActionCommand.java:76]: @AliciaGarciaGarcia It is not necessary to translate the message or add it to the language files. A simple message suffices.

As this is not something that important, can you remove the last two commits and forward the pr directly? I'm ok with all other changes and tests were green.  Sorry for the misunderstanding.

**Review by adolfopa — CHANGES_REQUESTED** [review]

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Comment by adolfopa** [comment]
ci:reevaluate:1297401_6589

**Pull request closed without merging**